docs/src/modules/components/MarkdownDocs.js

```diff
@@ -27,74 +28,82 @@ function MarkdownDocs(props) {
   const { description, location, rendered, title, toc, headers } = docs[userLanguage] || docs.en;
 
   return (
-    <AppLayoutDocs
-      description={description}
-      disableAd={disableAd}
-      disableToc={disableToc}
-      location={location}
-      title={title}
-      toc={toc}
-    >
-      {rendered.map((renderedMarkdownOrDemo, index) => {
-        if (typeof renderedMarkdownOrDemo === 'string') {
-          return <MarkdownElement key={index} renderedMarkdown={renderedMarkdownOrDemo} />;
-        }
-
-        if (renderedMarkdownOrDemo.component) {
-          const Component = markdownComponents[renderedMarkdownOrDemo.component];
-          return <Component key={index} headers={headers} options={renderedMarkdownOrDemo} />;
-        }
-
-        const name = renderedMarkdownOrDemo.demo;
-        const demo = demos?.[name];
-        if (demo === undefined) {
-          const errorMessage = [
-            `Missing demo: ${name}. You can use one of the following:`,
-            Object.keys(demos),
-          ].join('\n');
-
-          if (userLanguage === 'en') {
-            throw new Error(errorMessage);
+    <CodeCopyProvider>
```

Comment:
Maybe this is generic enough to be under the _app.js file?

Reply:
At first, I want to keep it to only some pages but I think you are right.